core/templates/dev/head/pages/admin-page/admin-page.constants.ts

```diff
@@ -16,25 +16,24 @@
  * @fileoverview Constants for the Oppia admin page.
  */
 
-var oppia = require('AppInit.ts').module;
-
-oppia.constant(
+angular.module('adminPageModule').constant(
   'ADMIN_ROLE_HANDLER_URL', '/adminrolehandler');
 
-oppia.constant('ADMIN_HANDLER_URL', '/adminhandler');
-oppia.constant(
+angular.module('adminPageModule').constant(
+  'ADMIN_HANDLER_URL', '/adminhandler');
+angular.module('adminPageModule').constant(
   'ADMIN_TOPICS_CSV_DOWNLOAD_HANDLER_URL', '/admintopicscsvdownloadhandler');
 
-oppia.constant(
+angular.module('adminPageModule').constant(
```

Comment:
This style is a little messy, since you keep repeating angular.module(XXX) again and again. Ideally can you do something like any of the examples here: https://github.com/johnpapa/angular-styleguide/tree/master/a1#ng-annotate

i.e.

```
angular.module('adminPageModule')
   .constant(..)
   .constant(...)
```

Reply:
Yes, that is a good idea. I have made the changes!
PTAL if they look good!

Reply:
Yes, thanks, it looks much better. Please make sure to do it everywhere it needs to be done, thanks!